packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx

```diff
@@ -21,18 +21,18 @@ const styles = {
   }),
 };
 
-interface WrappedEntryCardProps {
+export interface WrappedEntryCardProps {
   getEntityScheduledActions: SpaceAPI['getEntityScheduledActions'];
   getAsset: (assetId: string) => Promise<unknown>;
-  getEntryUrl?: (entryId: string) => string;
```

Comment:
@DanweDE Would this be a breaking change?

Reply:
Yes, but only for the `WrappedEntryCard`. One of the commits is marked as breaking and I've updated the RT editor accordingly.